ntc_templates/templates/cisco_ios_show_power_inline.textfsm

```diff
@@ -0,0 +1,28 @@
+Value Required INTERFACE (\S+)
+Value ADMIN (auto)
+Value OPER (on|off)
+Value POWER (\d+\.\d+)
+Value DEV_POWER (\d+\.\d+)
+Value DEVICE (.+?)
+Value CLASS (n\/a|\d)
+Value MAX (\d+\.\d)
+
+Start
+# Checking for modules headers
+  ^Module\s+Available\s+Used\s+Remaining\s*$$
+  ^Available:\d+(\.\d+)?\(w\)\s+Used:\d+(\.\d+)?\(w\)\s+Remaining:\d+(\.\d+)?\(w\)\s*$$
+  ^[\s(Wats)]*$$
+  ^[-\s]*$$
+  ^\d+\s+(n\/a|\d+\.\d+)\s+(n\/a|\d+\.\d+)\s+(n\/a|\d+\.\d+)\s*$$
+  ^Interface\s+Admin\s+Oper\s+Power(\(Watts\))?\s+Device\s+Class.*$$
```

Comment:
You can use the heading for the table to jump to new parsing section `^match -> Table`
this keep the header matching and the table parsing separate, less chance of matching something by accident.